pom.xml

```diff
@@ -40,6 +40,16 @@
       <artifactId>vertx-web</artifactId>
       <version>${vertx.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>folio-di-support</artifactId>
+      <version>1.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>folio-service-tools-dev</artifactId>
```

Comment:
I am unclear on the difference between folio-service-tools, folio-service-tools-dev and folio-service-tools-test. Thinking about it by the names, should this be folio-service-tools instead of folio-service-tools-dev?

Reply:
the `folio-service-tools` is the module containing two sub-modules `folio-service-tools-dev` - is the module containing classes such as exception handling commonly used in other modules and `folio-service-tools-test` - for common classes used for testing

Reply:
perfect, thanks for the explanation @natali-zaitseva